src/language-js/comments.js

```diff
@@ -787,17 +787,20 @@ function handleForComments({ comment, enclosingNode }) {
   return false;
 }
 
-function handleImportDeclarationComments({
+function handleImportExportDeclarationComments({
```

Comment:
This function seem only handle comments bettwen `ImportSpecifier`s and `ImportSpecifier`s, how about name it `handleSpecifierComments()`?

Reply:
`handleModuleSpecifiersComments` ? [13423fc](https://github.com/prettier/prettier/pull/12746/commits/13423fcdb86c63e1affb984d533d3abd31fc7b87)